src/app/features/activity-list/components/transactions/bitcoin-transaction.tsx

```diff
@@ -0,0 +1,11 @@
+import { BoxProps } from '@stacks/ui';
+
+import { BitcoinTransaction as BitcoinTx } from '@app/common/transactions/bitcoin/transaction.models';
+import { BitcoinTransactionItem } from '@app/components/transaction/bitcoin/bitcoin-transaction-item';
+
+interface BitcoinTransactionProps extends BoxProps {
+  transaction: BitcoinTx;
+}
+export const BitcoinTransaction = ({ transaction, ...rest }: BitcoinTransactionProps) => {
+  return <BitcoinTransactionItem transaction={transaction} {...rest} />;
+};
```

Comment:
What's the responsibility of this component?

Reply:
Yep, need to come back to this, thx.